qualang_tools/results/README.md

```diff
@@ -17,10 +17,10 @@ Then the results can be fetched with the `.fetch_all()` method while the program
 from qualang_tools.results import fetching_tool
 
 n_avg = 1000
-with program as prog:
+with program() as prog:
     # QUA program with n_avg averaging iterations
 
-qmm = QuantumMachinesManager(host="127.0.0.1", port="80")
+qmm = QuantumMachinesManager(qop_ip, cluster_name)

```

Comment:
The second argument of `QuantumMachinesManager` is still the port so I don't think this would work...
Maybe we can keep `qmm = QuantumMachinesManager(host=qop_ip, port=qop_port, cluster_name=cluster_name)`